Common/ImageSamplers/itkImageSamplerBase.hxx

```diff
@@ -301,6 +301,12 @@ ImageSamplerBase<TInputImage>::CropInputImageRegion()
    * InputImageRegion and the BoundingBoxRegion.
    */
   m_CroppedInputImageRegion = m_InputImageRegion;
+
+  if (m_ReduceDimension)
+  {
+    m_CroppedInputImageRegion.GetModifiableSize().back() = 1;
```

Comment:
@mstaring @stefanklein It does not seem to be necessary to also adjust the index (by doing `index[N-1] = 0`, maybe), but that depends on the use case, I guess... See https://github.com/SuperElastix/elastix/pull/1064#discussion_r1534075016